### PR TITLE
Main head title

### PR DESCRIPTION
**Pull request by Thiago-spart**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] I followed GitFlow pattern to create the branch
- [x] My code produces no warnings or errors in the console
- [x] Tests for the changes have been added / updated
- [x] Docs have been added / updated

## PR Type

What kind of change does this PR introduce?

```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Other... Please describe:
```

## What is the current behavior?
The old head wasn't using the HeadTitle component
## What is the new behavior?
The new has a better behavior and uses HeadTitle 
## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by henriqueleite42 — APPROVED** [review]

**Review by henriqueleite42 — DISMISSED** [review]

**Review by aleessandrohr — CHANGES_REQUESTED** [review]

File: src/assets/translations/en-US/home.json [hunk 1]
```diff
@@ -1,5 +1,8 @@
 {
 	"mainSection": {
+		"head": {
+			"title": "A Tech Community"
+		},
```
Comment (alessandrohenriqueramos) [inline src/assets/translations/en-US/home.json:5]: Acima da `mainSection`

File: src/assets/translations/pt-BR/home.json [hunk 2]
```diff
@@ -1,5 +1,8 @@
 {
 	"mainSection": {
+		"head": {
+			"title": "Uma Comunidade Tech"
+		},
```
Comment (alessandrohenriqueramos) [inline src/assets/translations/pt-BR/home.json:5]: Acima da `mainSection`

File: src/web/pages/Home/index.tsx [hunk 3]
```diff
@@ -8,16 +11,21 @@ import { TechmmunityProductsSection } from "./components/TechmmunityProductsSect
...
-		<Line />
-		<FeedbackSection />
-	</>
-);
+export const HomePage: FCWithLayout = () => {
+	const { t } = useTranslation("home");
+
+	return (
+		<>
+			<HeadTitle title={t("mainSection.head.title")} />
```
Comment (alessandrohenriqueramos) [inline src/web/pages/Home/index.tsx:19]: Atualizar a string

**Review by aleessandrohr — CHANGES_REQUESTED** [review]

File: src/assets/colors.ts [hunk 4]
```diff
@@ -1,4 +1,4 @@
-import { Colors as CoreColors } from "core/assets/colors";
+import { COLORS as CoreColors } from "core/assets/colors";
```
Comment (henriqueleite42) [inline src/assets/colors.ts:1]: `CoreColors` -> `CORE_COLORS`

**Review by henriqueleite42 — CHANGES_REQUESTED** [review]

**Review by henriqueleite42 — DISMISSED** [review]

**Review by aleessandrohr — APPROVED** [review]

**Review by henriqueleite42 — APPROVED** [review]

**Pull request merged**